src/main/java/org/parabot/randoms/oswar/ItemPicker.java

```diff
@@ -16,11 +15,12 @@
  */
 public class ItemPicker implements Random {
     private final int INTERFACE_ID = 33300;
-    private final String URL = "http://bdn.parabot.org/api/v2/data/items/";
 
     @Override
     public boolean activate() {
-        return Game.isLoggedIn() && Interfaces.getOpenInterfaceId() == INTERFACE_ID;
+        return Game.isLoggedIn()
+                && Interfaces.getOpenInterfaceId() == INTERFACE_ID
+                && !Interfaces.getInterface(INTERFACE_ID + 3).getMessage().contains("0:00");
```

Comment:
getMessage() probably need a null check

Reply:
That or I could try catch it, what would be best? However, I have been testing this and haven't had any problems yet, that's also why I didn't bother checking it.

Reply:
Add a null-check to it